internal/oidc/token/token_handler.go

```diff
@@ -187,6 +189,21 @@ func upstreamOIDCRefresh(ctx context.Context, session *psession.PinnipedSession,
 			WithDebugf("provider name: %q, provider type: %q", s.ProviderName, s.ProviderType))
 	}
 	if refreshedGroups != nil {
```

Comment:
er... what does it mean for `refreshedGroups` to be `nil`?

Reply:
For OIDC it means the claim doesn't exist.